xline/Cargo.toml

```diff
@@ -23,6 +23,7 @@ event-listener = "2.5.2"
 jsonwebtoken = "8.1.1"
 itertools = "0.10.3"
 utils = { path = "../utils", version = "0.1.0", features = ["parking_lot"] }
+engine = { path = "../engine", version = "0.1.0" }
```

Comment:
Actually I don't think we need a version, the path is enough. So does `utils` above.

Reply:
Fixed